content/en/_index.md

```diff
@@ -204,11 +237,18 @@ See https://cloud.google.com/compute/docs/quickstart-client-libraries before run
  * before running the code snippet.
  */
 {{< /tab >}}
+{{< tab header="Terraform" >}}
+// [START product_example]
```

Comment:
Other langauge tags do not include a region start or end tag

Reply:
fixed